skimage/measure/_colocalization.py

```diff
@@ -119,33 +118,35 @@ def manders_coloc_coeff(image0, image1_mask, mask=None):
 
     Notes
     -----
-    Manders' Colocalization Coefficient (MCC) is the fraction of total
-    intensity of a certain channel (channel A) that is within the segmented
-    region of a second channel (channel B) [1]_. It ranges from 0 for no
-    colocalisation to 1 for complete colocalization. It is also referred to
-    as M1 and M2.
+    Manders' colocalization coefficient (MCC) was developed in the context of
+    confocal fluorescence microscopy, to measure the fraction of colocalizing
```

Comment:
```suggestion
    confocal biological microscopy, to measure the fraction of colocalizing
```
@stefanv and here's another one ;)

Reply:
In practice, it is fluorescence microscopy (nowadays generating n-channel images, not just dual-channel). But the notes give some (historical) background about MCC, and the [historical paper](https://imagej.net/media/manders.pdf) precisely (Manders' paper) speaks of "dual-colour confocal microscopy" (detecting two fluorochromes, so it was fluorescence microscopy but I guess the term wasn't coined yet).

Replacing "fluorescence" with "biological" makes the subject not only more historical, but also more general. For people outside the field, it is informative that the technique applied to biological objects.

Reply:
Oh, maybe not 'historical' because I can see that Pawley's *Handbook of confocal fluorescence microscopy* (1995) has become *Handbook of confocal biological microscopy* in 2006! Maybe you can use tools other than fluorochromes in confocal microscopy... So I believe "confocal biological microscopy" is the better (more general) term here.

Reply:
Go ahead and push whatever version you prefer and merge!